src/context-menu-builder.js

```diff
@@ -21,10 +21,11 @@ export default class ContextMenuBuilder {
    * @param  {BrowserWindow|WebView} windowOrWebView  The hosting window/WebView
    * @param  {Boolean} debugMode    If true, display the "Inspect Element" menu item.
    */
-  constructor(spellCheckHandler, windowOrWebView=null, debugMode=false) {
+  constructor(spellCheckHandler, windowOrWebView=null, debugMode=false, additionalItems = []) {
```

Comment:
I'm not super excited about this API, can we make it a Function instead that gets a chance to edit the menu?